core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -175,7 +175,7 @@ class KafkaServer(val config: KafkaConfig, time: Time = Time.SYSTEM, threadNameP
    */
   def startup() {
     try {
-      info("starting")
+      info("Starting Kafka Server")
```

Comment:
Quick note: because of `logIdent`, there's no need to mention "Kafka Server", that will be in the prefix of the log message.

Reply:
Thanks @ijuma . Will update the log message.